.gitconfig

```diff
@@ -0,0 +1,3 @@
+[user]
+	name = Lucas Rose
+	email = lrose@qualtrax.com
```

Comment:
This shouldnt be in the repo it should be in your user folder. These two commands will configure it for you:

```
git config --global user.name "Lucas Rose"
git config --global user.email "lrose@qualtrax.com"
```